static/templates/admin/addFileForm.html

```diff
@@ -0,0 +1,23 @@
+<iframe id="upload_file_frame" name="upload_file_frame" height="0" width="0" style="display: none;"></iframe>
+<form id="create_simple_object_form" method="post" action="ingest/<%= pid %>" enctype="multipart/form-data" target="upload_file_container" class="edit_form create_simple_object_form">
+    <p>Add a single file as a new object.  If no label is provided, the filename will be used.</p>
```

Comment:
Should probably read "Add a file to this aggregate object" or "Add a file to this work" if you want to future proof it